gammapy/image/core.py

```diff
@@ -298,6 +298,39 @@ def contains(self, position):
         x, y = skycoord_to_pixel(position, self.wcs, self.wcs_origin)
         return (x >= 0.5) & (x <= nx + 0.5) & (y >= 0.5) & (y <= ny + 0.5)
 
+    def footprint(self, mode='corners'):
+        """
+        Calculates the footprint of the image on the sky.
+
+        Calls `~astropy.wcs.WCS.calc_footprint`, passing ``mode`` and ``kwargs`` to it.
+
+        Parameters
+        ----------
+        mode : {'centers', 'corners'}
+            Return the positions of the corners of the image on the sky by
+            using the center of the pixel or the corner.
+
+        Returns
+        -------
+        coordinates : `~collections.OrderedDict`
+            Dictionary of the positions of the corners of the image
+            with keys {'lower left', 'lower right', 'upper left', 'upper right'}
+            and `~astropy.coordinates.SkyCoord` objects as values.
+
+        TODO: Examples?
```

Comment:
Yes, having a simple example for most functions / methods is good.
E.g. I know when I read the docstring for some Numpy function, often it's the examples that help me understand what the inputs and outputs are.

For this one, you could print the full return value, or if it is too long, just one or two entries in the dict.